include/deal.II/gmsh/utilities.h

```diff
@@ -94,6 +104,60 @@ namespace Gmsh
     Triangulation<2, spacedim> &tria,
     const AdditionalParameters &prm = AdditionalParameters());
 #  endif
+
+#  ifdef DEAL_II_GMSH_WITH_API
+  /**
+   * Read a partitioned GMSH mesh file and create a fully distributed
+   * triangulation in deal.II.
+   *
+   * This function reads partitioned mesh files generated by GMSH's internal
+   * partitioner (usually metis) and constructs a fully distributed
+   * triangulation that can be used in parallel computations with MPI. The
+   * function automatically handles ghost cell identification, vertex
+   * coordinates, and cell connectivity across processor boundaries.
+   *
+   * In order to generate a set of partitioned mesh files that can be used with
+   * this function, you could use the `GMSH` command line tool with the
+   * following options:
+   * @code
+   * gmsh -part 4 -part_ghosts -part_split -save {input_file}
+   * @endcode
+   *
+   * This will generate a set of mesh files named like `{input_file}_1.msh`,
+   * `{input_file}_2.msh`, etc.
```

Comment:
The naming of these files is perhaps slightly confusing because it's the *output* from the perspective of the program you're calling here :-) Maybe this?
```suggestion
   * gmsh -part 4 -part_ghosts -part_split -save {msh_file_base}
   * @endcode
   *
   * This will generate a set of mesh files named like `{msh_file_base}_1.msh`,
   * `{msh_file_base}_2.msh`, etc.
```